guides/tls-certificates/cloudDNS.md

```diff
@@ -1,16 +1,19 @@
 ---
-title: Cloud DNS
+title: Google Cloud DNS
 description:
-  Learn how to use cert-manager to set up SSL certificates using Cloud DNS for
-  DNS01 challenges.
+  Learn how to use cert-manager to set up TLS certificates using Google Cloud
+  DNS for DNS01 challenges.
 ---
 
 [cert-manager](https://cert-manager.io/) allows you to enable HTTPS on your
 Coder installation, regardless of whether you're using
 [Let's Encrypt](https://letsencrypt.org/) or you have your own certificate
 authority.
 
-This guide will show you how to install cert-manager v1.0.1 and set up your
+> Note: This guide is for Coder v1.21 which has a slightly different way of
+> handling certificates. Ensure you are reading the docs for your Coder version.
```

Comment:
`This guide is for Coder v1.21.0 and later, which handle certificates differently from earlier versions of Coder. Ensure that you're reading the docs applicable to your Coder version.`